test/e2e/framework/framework.go

```diff
@@ -45,7 +45,6 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
-
 	// required to use gcloud login see: https://github.com/kubernetes/client-go/issues/242
```

Comment:
[Linter](https://pantheon.corp.google.com/cloud-build/builds/f0bf82cd-9367-4c9b-b0c6-970ce8dfc421;step=13?e=13803378&mods=logs_tg_prod&project=agones-images) is complaining about the removed blank line. Can you add it back and retry?

Reply:
I tried it, lint complains:
```
build/scripts/feature-shortcode-update/main.go:1: : error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping. (typecheck)
build/scripts/remove-data-proofer-ignore/main.go:1: : error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping. (typecheck)
build/scripts/sdk-update-version/main.go:1: : error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping. (typecheck)
```

Reply:
I've included this `-buildvcs=false` flag but getting the same error.